inference-engine/thirdparty/clDNN/api/tensor.hpp

```diff
@@ -189,6 +189,8 @@ struct format {
         g_is_os_zyx_osv16_isv16,                      ///< format used for grouped weights for blocked 3D deconvolution
         g_os_is_yx_osv16_isv4,
         g_is_os_yx_osv16_isv16,
+        os_is_zyx_osv16_isv16,
+        g_os_is_zyx_osv16_isv16,        
```

Comment:
As I can see you haven't added these layouts into format_traits map

Reply:
Added, and also fixed incorrect X, Y order in first string of 'os_is_yx_osv16_isv4' and 'g_os_is_yx_osv16_isv4' formats.